CMakeLists.txt

```diff
@@ -266,24 +274,43 @@ if (viam_rust_utils_files)
     ${viam_rust_utils_file}
     ONLY_IF_DIFFERENT
   )
-elseif(NOT WIN32) # TODO(RSDK-10366): Currently, rust_utils is not published for windows, so don't even try downloading
+elseif(NOT WIN32 OR rust_utils_supported_windows_arch) # TODO(RSDK-10366): Currently, rust_utils is not published for windows aarch, so don't even try downloading
   set(lvru_system_name ${CMAKE_SYSTEM_NAME})
-  if (CMAKE_SYSTEM_NAME STREQUAL "Darwin")
+  if (CMAKE_SYSTEM_NAME STREQUAL "Cygwin")
```

Comment:
Is supporting cygwin something important?

Reply:
Not that I'm aware of, I wasn't sure so I was trying to be conservative.